client/lib/plans/plans-list.js

```diff
@@ -279,7 +279,7 @@ const getPlanBusinessDetails = () => ( {
 	getStoreAudience: () => i18n.translate( 'The plan for small businesses' ),
 	getDescription: () =>
 		i18n.translate(
-			'{{strong}}Best for Small Businesses:{{/strong}} Power your' +
+			'{{strong}}Best for small businesses:{{/strong}} Power your' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( '{{strong}}Best for Small Businesses:{{/strong}} Power your business website with custom plugins and themes, unlimited premium and business theme templates, 200 GB storage, and the ability to remove WordPress.com branding.' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/40107-sentence-case)